src/cascadia/TerminalSettingsModel/IAppearanceConfig.idl

```diff
@@ -22,6 +22,14 @@ namespace Microsoft.Terminal.Settings.Model
         Vertical_Bottom = 0x20
     };
 
+    [flags]
+    enum IntenseStyle
+    {
+        Bold = 0x1,
+        // Bright = 0x2,

```

Comment:
might as well leave this one uncommented. after all, it's only accessible if you add it to the enum mapper